

# **Understanding and analyzing the distribution of youth poverty population across the US in 2022.**

## **Introduction:**

  

      

Socioeconomic landscape of the United States presents a complex and multifaceted picture, particularly when viewed through the lens of household income and child poverty. Analyzing data from various states and school districts reveals significant disparities in median household incomes, which range from as low as  28,972 to as high as 167,605, with an average of 63,468. This wide income variability, coupled with a substantial standard deviation of 16,308.75, underscores the economic inequities that pervade the nation. The plight of children living in poverty is starkly illustrated by the staggering total of 6,664,873 children aged 5-17 affected, averaging about 1,015.99 per state but with a notable standard deviation of 5,529.49, highlighting the uneven distribution of poverty. This analysis also identified specific school districts, such as the New York City Department of Education, Puerto Rico, and Chicago Public School District 299, where child poverty is alarmingly concentrated. Furthermore, a detailed examination of Texas, the state with the largest total population, revealed that while the overall population is substantial, around one million children live in poverty, representing 3.4% of the state's total population. This data-driven exploration emphasizes the urgent need for targeted policies and interventions to address the economic disparities and support the nation's most vulnerable populations, particularly children.

### **Objectives:**

     

- Identify  trends in childhood poverty within the United States.
- To analyze the socio-economic effects of poverty on school Districts 
  and communities(County wide).
- To evaluate State and County income levels of childhood poverty .
- To propose evidence-based solutions to reduce poverty.

### **Research Questions:**

- What is the median income  in each state, and how does it correlate with the number of children in poverty?
- What Statistical data can we gather from this data?
- What is the distribution of youth poverty distributed across various school districts?
- What State as the largest total Population and what percentage of children live in poverty?
- Compare different state's  youth population within the United States whom live in poverty?

## **Methodology:**

    


###       **1.Data Collection**

##### **1.1 Data Acquisition**

The data acquisition process used within this project  focused on retrieving data from the US Census Bureau and storing those data 

points collected in a Microsoft SQL server database for later analysis.These datasets housed different population characteristics of 

states and school districts and income within the United States for the year 2022.Such data included the total population and estimates 

of youth between the ages of 5 and 17 living in poverty in school districts, counties, and states.

##### **1.2 Data Description**

The US School District dataset contains demographic and socio-economic data for school districts across the United States 

for the year 2022.Each record provides information such as the state postal code, state FIPS code, district ID, school district

name, and estimated total population. It also includes specific data onthe estimated population of children aged 5-17 and the 

number of children in poverty. This dataset is crucial for educational researchers and policymakers to understand the demographic 

landscape of school districts and to address issues related to educational inequality and resource distribution.The 2022 Income Estimate 

dataset provides detailed poverty and income statistics across various states and counties in the United States. Each record includes 

data such as state and county FIPS codes, postal codes, poverty estimates and percentages for different age groups, and median household

income. The dataset is essential for socio-economic research, enabling analysts to assess poverty levels, identify areas in need of economic 

support, and evaluate the effectiveness of policies aimed at reducing poverty and improving income levels across different regions.

###       **2.Data Exploration**

Within the data Exploration process data from the US Census Bureau, which including fields such as State Postal Code, State FIPS Code, 

District ID, School District, Estimated Total Population, Estimated Population 5-17, and the number of children aged 5-17 living in poverty. 

Initially, the dataset was inspected by counting the number of rows to understand the dataset's size. The table structure was then reviewed 

to confirm the presence and types of columns.A sample of the data was viewed to get a preliminary sense of its content and quality, 

checking for any obvious inconsistencies or missing values. This initial exploration provided a foundational understanding of the 

dataset, essential for subsequent detailed analysis.

#####  **2.1 View Table Structure:**

In [1]:
-- Check the structure of the '2022_income_Estimate' table
EXEC sp_help '2022_income_Estimate';

The object '2022_income_Estimate' does not have any indexes, or you do not have permissions.

No constraints are defined on object '2022_income_Estimate', or you do not have permissions.

No foreign keys reference table '2022_income_Estimate', or you do not have permissions on referencing tables.

No views with schema binding reference table '2022_income_Estimate'.

Total execution time: 00:00:05.076

Name Owner Type Created_datetime 2022_income_Estimate dbo user table 2024-06-26 17:13:31.030

Column_name Type Computed Length Prec Scale Nullable TrimTrailingBlanks FixedLenNullInSource Collation State_FIPS_Code tinyint no 1 3 0 yes (n/a) (n/a) NULL County_FIPS_Code smallint no 2 5 0 yes (n/a) (n/a) NULL Postal_Code nvarchar no 100 yes (n/a) (n/a) SQL_Latin1_General_CP1_CI_AS Name nvarchar no 100 yes (n/a) (n/a) SQL_Latin1_General_CP1_CI_AS Poverty_Estimate_All_Ages int no 4 10 0 yes (n/a) (n/a) NULL Poverty_Percent_All_Ages float no 8 53 NULL yes (n/a) (n/a) NULL Poverty_Estimate_Age_0_17 int no 4 10 0 yes (n/a) (n/a) NULL Poverty_Percent_Age_0_17 float no 8 53 NULL yes (n/a) (n/a) NULL Poverty_Estimate_Age_5_17_in_Families int no 4 10 0 yes (n/a) (n/a) NULL Poverty_Percent_Age_5_17_in_Families float no 8 53 NULL yes (n/a) (n/a) NULL Median_Household_Income int no 4 10 0 yes (n/a) (n/a) NULL

Identity Seed Increment Not For Replication No identity column defined. NULL NULL NULL

RowGuidCol No rowguidcol column defined.

Data_located_on_filegroup PRIMARY

In [2]:
-- Check the structure of the 'USSchoolDistrict' table
EXEC sp_help 'USSchoolDistrict';




The object 'USSchoolDistrict' does not have any indexes, or you do not have permissions.

No constraints are defined on object 'USSchoolDistrict', or you do not have permissions.

No foreign keys reference table 'USSchoolDistrict', or you do not have permissions on referencing tables.

No views with schema binding reference table 'USSchoolDistrict'.

Total execution time: 00:00:01.248

Name Owner Type Created_datetime USSchoolDistrict dbo user table 2024-06-23 16:59:02.347

Column_name Type Computed Length Prec Scale Nullable TrimTrailingBlanks FixedLenNullInSource Collation State_Postal_Code nvarchar no 100 yes (n/a) (n/a) SQL_Latin1_General_CP1_CI_AS State_FIPS_Code tinyint no 1 3 0 yes (n/a) (n/a) NULL District_ID varchar no 500 yes no yes SQL_Latin1_General_CP1_CI_AS School_District nvarchar no 1000 yes (n/a) (n/a) SQL_Latin1_General_CP1_CI_AS Estimated_Total_Population float no 8 53 NULL yes (n/a) (n/a) NULL Estimated_Population_5_17 float no 8 53 NULL yes (n/a) (n/a) NULL ChildrenInPoverty float no 8 53 NULL yes (n/a) (n/a) NULL Total_State_Population int no 4 10 0 yes (n/a) (n/a) NULL Total_Youth_Poverty_By_State int no 4 10 0 yes (n/a) (n/a) NULL

Identity Seed Increment Not For Replication No identity column defined. NULL NULL NULL

RowGuidCol No rowguidcol column defined.

Data_located_on_filegroup PRIMARY

#####        **2.2 View Sample Data**

In [3]:
-- View the first 10 rows of the '2022_income_Estimate' table
SELECT TOP 10 * FROM [dbo].[2022_income_Estimate];

(10 rows affected)

Total execution time: 00:00:00.295

State_FIPS_Code County_FIPS_Code Postal_Code Name Poverty_Estimate_All_Ages Poverty_Percent_All_Ages Poverty_Estimate_Age_0_17 Poverty_Percent_Age_0_17 Poverty_Estimate_Age_5_17_in_Families Poverty_Percent_Age_5_17_in_Families Median_Household_Income 1 0 AL Alabama 798469 16.200000762939453 237861 21.799999237060547 165972 20.600000381469727 59703 1 1 AL Autauga County 6988 11.800000190734863 2151 15.699999809265137 1528 14.800000190734863 70148 1 3 AL Baldwin County 30195 12.399999618530273 8093 16.100000381469727 5341 14 71704 1 5 AL Barbour County 5860 26.700000762939453 1871 37.70000076293945 1244 33.900001525878906 41151 1 7 AL Bibb County 3979 20 1084 25.5 805 25.799999237060547 54309 1 9 AL Blount County 8022 13.600000381469727 2105 15.800000190734863 1513 15.199999809265137 60553 1 11 AL Bullock County 2670 31.5 825 39 588 37.29999923706055 35798 1 13 AL Butler County 4298 23.399999618530273 1360 33 1007 33.099998474121094 41852 1 15 AL Calhoun County 20308 18.5 6257 26.200000762939453 4284 24.399999618530273 52772 1 17 AL Chambers County 6962 21 2097 30.600000381469727 1574 30.899999618530273 45563

In [4]:
-- View the first 10 rows of the 'USSchoolDistrict' table
SELECT TOP 10 * FROM USSchoolDistrict;

(10 rows affected)

Total execution time: 00:00:00.370

State_Postal_Code State_FIPS_Code District_ID School_District Estimated_Total_Population Estimated_Population_5_17 ChildrenInPoverty Total_State_Population Total_Youth_Poverty_By_State DE NULL 00080 Appoquinimink School District 70402 14158 735 1018396 20639 DE NULL 01240 Brandywine School District 92915 13765 1464 1018396 20639 DE NULL 00180 Caesar Rodney School District 49824 8893 1143 1018396 20639 DE NULL 00170 Cape Henlopen School District 66441 6558 679 1018396 20639 DE NULL 00190 Capital School District 60983 9399 1865 1018396 20639 DE NULL 00200 Christina School District 176806 25839 3970 1018396 20639 DE NULL 00230 Colonial School District 89368 14178 2156 1018396 20639 DE NULL 00270 Delmar School District 7795 1434 203 1018396 20639 DE NULL 00680 Indian River School District 99649 11669 1768 1018396 20639 DE NULL 00790 Lake Forest School District 26971 4543 634 1018396 20639

##### **2.3 Count the Number of rows**

In [5]:
-- Count the number of rows in the 'USSchoolDistrict' table
SELECT COUNT(*) FROM USSchoolDistrict;

(1 row affected)

Total execution time: 00:00:00.020

(No column name) 11224

In [6]:
-- Count the number of rows in the '2022_income_Estimate' table
SELECT COUNT(*) FROM dbo.[2022_income_Estimate];

(1 row affected)

Total execution time: 00:00:00.006

(No column name) 3195

##### **2.4 Descriptive Statistics**

In [7]:
-- Calculate basic statistics for the 'Estimated_Total_Population' column
SELECT 
  AVG(Estimated_Total_Population) AS avg_value,
  MIN(Estimated_Total_Population) AS min_value,
  MAX(Estimated_Total_Population) AS max_value,
  COUNT(Estimated_Total_Population) AS total_count
FROM 
  USSchoolDistrict;

(1 row affected)

Total execution time: 00:00:00.020

avg_value min_value max_value total_count 25002.028510334996 0 8335897 11224

In [8]:
-- Calculate basic statistics for the 'Estimated_Population_5_17' column
SELECT 
  AVG(Estimated_Population_5_17) AS avg_value,
  MIN(Estimated_Population_5_17) AS min_value,
  MAX(Estimated_Population_5_17) AS max_value,
  COUNT(Estimated_Population_5_17) AS total_count
FROM 
  USSchoolDistrict;

(1 row affected)

Total execution time: 00:00:00.019

avg_value min_value max_value total_count 3905.5225409836066 0 1205959 11224

In [9]:
-- Calculate basic statistics for the 'Median_Household_Income' column
SELECT 
  AVG(Median_Household_Income) AS avg_value,
  MIN(Median_Household_Income) AS min_value,
  MAX(Median_Household_Income) AS max_value,
  COUNT(Median_Household_Income) AS total_count
FROM 
  [dbo].[2022_income_Estimate];

Warning: Null value is eliminated by an aggregate or other SET operation.

(1 row affected)

Total execution time: 00:00:00.067

avg_value min_value max_value total_count 63468 28972 167605 3194

In [10]:
-- Count the number of rows for each 'State_Postal_Code'
SELECT 
  State_Postal_Code, 
  COUNT(State_Postal_Code) AS count
FROM 
  USSchoolDistrict
GROUP BY 
  State_Postal_Code;

(45 rows affected)

Total execution time: 00:00:00.171

State_Postal_Code count TX 1018 KS 286 PA 500 WI 421 DE 16 IN 290 IL 851 NH 179 MD 24 DC 1 MI 540 VA 136 ME 267 SC 75 RI 36 WA 295 MA 294 NV 17 NY 680 OH 611 SD 149 LA 69 MO 516 NJ 557 OR 197 MN 329 MT 399 OK 509 PR 1 FL 67 IA 327 VT 51 ND 172 ID 115 GA 182 MS 137 TN 141 NC 118 NE 244 WV 55 WY 48 HI 1 NM 89 KY 173 UT 41

### **3<mark></mark>.Data Preprocessing**

Data Preprocessing involved several critical steps to ensure the dataset's accuracy and completeness. First, missing and null values were

identified and addressed ,Duplicates were identified and removed to prevent skewed analysis results. Finally, the dataset was reviewed to 

confirm that all entries were accurate and consistent, setting the stage for reliable and meaningful analysis.

 ##### **3.1 Handling Missing and NULL values**

In [11]:
-- Check for missing values in specified columns
SELECT 
  'State_Postal_Code' AS ColumnName, 
  COUNT(*) - COUNT(State_Postal_Code) AS MissingValues
FROM 
  USSchoolDistrict
UNION ALL
SELECT 
  'Estimated_Population_5_17' AS ColumnName, 
  COUNT(*) - COUNT(Estimated_Population_5_17) AS MissingValues
FROM 
  USSchoolDistrict
UNION ALL
SELECT 
  'Estimated_Total_Population' AS ColumnName, 
  COUNT(*) - COUNT(Estimated_Total_Population) AS MissingValues
FROM 
  USSchoolDistrict;


(3 rows affected)

Total execution time: 00:00:00.081

ColumnName MissingValues State_Postal_Code 0 Estimated_Population_5_17 0 Estimated_Total_Population 0

##### **3.2 Handling Duplicate values**

In [12]:
-- Check for duplicates in dbo.2022_income
WITH DuplicateCheck_income AS (
    SELECT 
        State_FIPS_Code,
        Postal_Code,
        County_FIPS_Code,
        Median_Household_Income,
        Poverty_Estimate_Age_5_17_in_Families,
        COUNT(*) AS cnt
    FROM 
        [dbo].[2022_income_Estimate]
    GROUP BY 
        State_FIPS_Code,
        Postal_Code,
        County_FIPS_Code,
        Median_Household_Income,
        Poverty_Estimate_Age_5_17_in_Families
    HAVING 
        COUNT(*) > 1
)
SELECT 
    State_FIPS_Code,
    Postal_Code,
    County_FIPS_Code,
    Median_Household_Income,
    Poverty_Estimate_Age_5_17_in_Families,
    cnt
FROM 
    DuplicateCheck_income;


(0 rows affected)

Total execution time: 00:00:00.098

State_FIPS_Code Postal_Code County_FIPS_Code Median_Household_Income Poverty_Estimate_Age_5_17_in_Families cnt

##### **3.4 Delete unwanted Rows**

In [13]:

-- Delete a specific row based on a condition
DELETE FROM [2022_income_Estimate]
WHERE Postal_Code = 'US' AND Median_Household_income = 74755; 


(0 rows affected)

Total execution time: 00:00:00.054

### **4.Feature Engineering**

Feature engineering plays a crucial role in enhancing the predictive power and interpretability of the models used to analyze youth poverty across the United States. Key steps include creating new features from the existing data columns, such as calculating the poverty rate as the ratio of the number of children aged 5-17 in poverty to the total population of that age group in each school district. Additionally, geographical features like state-specific dummy variables can capture regional effects. Temporal features can be engineered to identify trends over time if historical data is available. Interaction terms between demographic variables, such as the interaction between the total population and the number of children in poverty, will be created to capture complex relationships. Socio-economic indicators like median household income and unemployment rates will be integrated to provide a broader context for poverty levels. These engineered features will help in building more accurate and insightful models for understanding and addressing youth poverty.

##### **3.1 Data Transformation**

In [14]:
-- drop a column
ALTER TABLE USSchoolDistrict DROP COLUMN Percentage_Of_Youth ;

: Msg 4924, Level 16, State 1, Line 2
ALTER TABLE DROP COLUMN failed because column 'Percentage_Of_Youth' does not exist in table 'USSchoolDistrict'.

Total execution time: 00:00:00.582

In [32]:
-- Drop unnecessary columns from 'USSchoolDistrict' table
ALTER TABLE USSchoolDistrict DROP COLUMN Total_Youth_Homeless_By_State ;

Commands completed successfully.

Total execution time: 00:00:00.241

In [38]:
-- Drop unnecessary columns from 'USSchoolDistrict' table
ALTER TABLE USSchoolDistrict DROP COLUMN Total_Youth_Poverty_By_State ;

Commands completed successfully.

Total execution time: 00:00:00.117

In [4]:
-- change column name
EXEC sp_rename 'dbo.USSchoolDistrict.Estimated_number_of_relevant_children_5_to_17_years_old_in_poverty_who_are_related_to_the_householder', 'ChildrenInPoverty', 'COLUMN';



Caution: Changing any part of an object name could break scripts and stored procedures.

Total execution time: 00:00:01.118

In [39]:
-- add a column and alter the table
ALTER TABLE USSchoolDistrict 
ADD Total_Youth_Poverty_By_State INT;


Commands completed successfully.

Total execution time: 00:00:00.288

In [41]:
-- Create an temp result set(StatePovertySum) to update the Total_Youth_Poverty_BY_State column
WITH StatePovertySum AS (
    SELECT 
        [State_Postal_Code],
        SUM([ChildrenInPoverty]) AS Total_Children_In_Poverty
    FROM 
        USSchoolDistrict
    GROUP BY 
        [State_Postal_Code]
)
UPDATE USSchoolDistrict
SET Total_Youth_Poverty_BY_State = (
    SELECT Total_Children_In_Poverty
    FROM StatePovertySum
    WHERE StatePovertySum.[State_Postal_Code] = USSchoolDistrict.[State_Postal_Code]
);




(11224 rows affected)

Total execution time: 00:00:01.104

In [44]:
-- view added data from the result set above
SELECT State_Postal_Code,Total_Youth_Poverty_By_State
FROM dbo.USSchoolDistrict;

(11224 rows affected)

Displaying Top 5000 rows.

Total execution time: 00:00:00.342

State_Postal_Code Total_Youth_Poverty_By_State DE 20639 DE 20639 DE 20639 DE 20639 DE 20639 DE 20639 DE 20639 DE 20639 DE 20639 DE 20639 DE 20639 DE 20639 DE 20639 DE 20639 DE 20639 DE 20639 DC 17274 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 FL 500989 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 GA 301323 HI 24567 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 ID 38776 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301563 IL 301

In [16]:
-- Find the school district with the largest amount of children
SELECT 
    State_Postal_Code, 
    School_District, 
    Estimated_Population_5_17
FROM 
    USSchoolDistrict
WHERE 
    Estimated_Population_5_17 = (
        SELECT 
            MAX(Estimated_Population_5_17)
        FROM 
            USSchoolDistrict
    );


(1 row affected)

Total execution time: 00:00:00.442

State_Postal_Code School_District Estimated_Population_5_17 NY New York City Department Of Education 1205959

### 4.**Visualization of the Data**

##### 4.1 What is the median income  in each state, and how does it correlate with the number of children in poverty?

 States such as Wyoming (WY), Washington (WA), and Virginia (VA) have higher median household incomes, while states including  New Mexico (NM) and Louisiana (LA) show lower median household incomes. The poverty estimates vary across states, with some states like Colorado (CO) having a higher number of children in poverty compared to others. This visualization helps to identify the disparity between income levels and poverty rates among different states.

In [16]:

WITH MaxIncomeCTE AS (
    SELECT 
        Postal_Code,
        MAX(Median_Household_income) AS Max_Median_Household_income
    FROM 
        [2022_income_Estimate]
    GROUP BY 
        Postal_Code
)

SELECT 
    e.Postal_Code,e.Median_Household_Income,e.Poverty_Estimate_Age_0_17
FROM 
    [2022_income_Estimate] e
JOIN 
    MaxIncomeCTE m
ON 
    e.Postal_Code = m.Postal_Code
    AND e.Median_Household_income = m.Max_Median_Household_income;


(52 rows affected)

Total execution time: 00:00:05.291

Postal_Code Median_Household_Income Poverty_Estimate_Age_0_17 WY 127677 208 WV 87259 1417 WI 99525 4040 WA 116044 38828 VT 86579 2572 VA 167605 4278 UT 132358 484 TX 124291 1909 TN 133486 2380 SD 98387 1026 SC 83692 6993 RI 101158 606 PA 117326 7521 OR 98706 10991 OK 81761 4201 OH 121528 2494 NY 135528 19010 NV 91960 814 NM 139217 117 NJ 137334 1042 NH 107445 2789 NE 98619 3386 ND 90092 152 NC 97099 21956 MT 83697 231 MS 80641 3649 MO 98925 5350 MN 117602 1085 MI 101227 2039 ME 88571 3640 MD 133068 4856 MA 118494 24170 LA 81284 4133 KY 116173 641 KS 99174 6700 IN 117068 3619 IL 102509 2573 ID 98707 350 IA 109138 1401 HI 96304 21616 GA 130909 2701 FL 103757 3933 DE 85427 15976 DC 99897 24651 DC 99897 24651 CT 122080 11348 CO 140768 2053 CA 150502 25899 AZ 83668 139817 AR 86374 7444 AL 98263 3939 AK 112218 38

##### 4.2 What Statistical data can we gather from this data?

##### Interpretation of statistical data using a table:

- The average median household income is 63,468, with significant variation as indicated by the standard deviation of 16,308.75. This suggests that while many households have incomes around the average, there are also many households with significantly higher or lower incomes.
- The minimum and maximum values for median household income indicate a wide range of economic conditions across different areas, from 28,972 to 167,605.
- The total number of children aged 5-17 living in poverty is substantial at 6,664,873. The average number of children in poverty per state is about 1,016, but this number varies widely, with a standard deviation of 5,529.49.
- The range of children in poverty per state is also very wide, with some states having as few as 2 children in poverty and others having up to 292,262.

These statistics provide a detailed snapshot of the economic conditions and the prevalence of poverty among children aged 5-17 across the data set.

In [6]:
SELECT 
    'Mean_Median_Household_Income' AS Statistic,
    AVG(Median_Household_income) AS Value
FROM 
    [dbo].[2022_income_Estimate]
WHERE 
    Median_Household_income >= 100

UNION ALL

SELECT 
    'StdDev_Median_Household_Income' AS Statistic,
    STDEV(Median_Household_income) AS Value
FROM 
   [dbo].[2022_income_Estimate]
WHERE 
    Median_Household_income >= 100

UNION ALL

SELECT 
    'Min_Median_Household_Income' AS Statistic,
    MIN(Median_Household_income) AS Value
FROM 
    [dbo].[2022_income_Estimate]
WHERE 
    Median_Household_income NOT IN (0, 1)

UNION ALL

SELECT 
    'Max_Median_Household_Income' AS Statistic,
    MAX(Median_Household_income) AS Value
FROM 
   [dbo].[2022_income_Estimate]
WHERE 
    Median_Household_income NOT IN (0, 1)

UNION ALL

SELECT 
    'Total_Poverty_Estimate_Age_5_17' AS Statistic,
    SUM(ChildrenInPoverty) AS Value
FROM 
    USSchoolDistrict
WHERE 
    ChildrenInPoverty >= 100

UNION ALL

SELECT 
    'Mean_Poverty_Estimate_Age_5_17' AS Statistic,
    AVG(ChildrenInPoverty) AS Value
FROM 
    USSchoolDistrict
WHERE 
    ChildrenInPoverty >= 100

UNION ALL

SELECT 
    'StdDev_Poverty_Estimate_Age_5_17' AS Statistic,
    STDEV(ChildrenInPoverty) AS Value
FROM 
    USSchoolDistrict
WHERE 
    ChildrenInPoverty >= 100

UNION ALL

SELECT 
    'Min_Poverty_Estimate_Age_5_17' AS Statistic,
    MIN(ChildrenInPoverty) AS Value
FROM 
    USSchoolDistrict
WHERE 
    ChildrenInPoverty NOT IN (0, 1)

UNION ALL

SELECT 
    'Max_Poverty_Estimate_Age_5_17' AS Statistic,
    MAX(ChildrenInPoverty) AS Value
FROM 
    USSchoolDistrict
WHERE 
    ChildrenInPoverty NOT IN (0, 1);



##### 4.3 What School Districts had the highest amount of children in Poverty

This chart displays the US school districts with the highest number of children in poverty. The New York City Department of Education has the highest number of children in poverty, followed by Puerto Rico and the Chicago Public School District 299. Other notable districts with high numbers of children in poverty include the Houston Independent School District, Clark County School District, Philadelphia City School District, Dade County School District, Detroit Public Schools Community District, Broward County School District, and Dallas Independent School District. This chart highlights the significant concentration of child poverty in these major school districts across the United States.

In [10]:
SELECT TOP 10
    School_District,
    SUM(ChildrenInPoverty) AS Total_ChildrenInPoverty
FROM 
    USSchoolDistrict
GROUP BY 
    School_District
ORDER BY 
    Total_ChildrenInPoverty DESC;


(10 rows affected)

Total execution time: 00:00:00.484

School_District Total_ChildrenInPoverty New York City Department Of Education 292262 Puerto Rico 233738 Chicago Public School District 299 91199 Houston Independent School District 69174 Clark County School District 65071 Philadelphia City School District 64906 Dade County School District 61434 Detroit Public Schools Community District 53485 Broward County School District 52357 Dallas Independent School District 42452

##### 4.4 What State as the largest total Population and what percentage of children live in poverty?

The pie chart illustrates the population distribution in the state of Texas, which has the largest total population among all states. The segments of the pie chart represent different population metrics:

- **Total\_Estimated\_Population (Blue):** This is the largest segment, indicating that the total estimated population of Texas constitutes the most significant proportion compared to the other metrics.
- **Total\_ChildrenInPoverty (Yellow):** This smaller segment shows the number of children living in poverty within Texas.
- **Total\_Estimated\_Population\_5\_17 (Gray):** This segment represents the estimated population of children aged 5-17 in Texas.

The chart visually demonstrates that the overall population is dominated by the total estimated population, while the portions representing children in poverty and the population aged 5-17 are relatively smaller. This indicates that while the population of children and those in poverty is significant, they are smaller compared to the total population of the state. Of the estimated total population of around thirty million, about one million children between the ages of 5 to 17 live in poverty, representing approximately 3.4 percent of the total population of the state of Texas living in poverty.


In [12]:
WITH StatePopulationSums AS (
    SELECT 
        State_Postal_Code, 
        SUM(Estimated_Total_Population) AS Total_Estimated_Population,
        SUM(ChildrenInPoverty) AS Total_ChildrenInPoverty,
        SUM(Estimated_Population_5_17) AS Total_Estimated_Population_5_17
    FROM 
        USSchoolDistrict
    GROUP BY 
        State_Postal_Code
)


SELECT 
    State_Postal_Code, 
    Total_Estimated_Population,
    Total_ChildrenInPoverty,
    Total_Estimated_Population_5_17
FROM 
    StatePopulationSums
WHERE 
    Total_Estimated_Population = (
        SELECT 
            MAX(Total_Estimated_Population)
        FROM 
            StatePopulationSums
    );


(1 row affected)

Total execution time: 00:00:00.231

State_Postal_Code Total_Estimated_Population Total_ChildrenInPoverty Total_Estimated_Population_5_17 TX 30030370 1002891 5553699

##### 4.5 Compare different state's  youth population within the United States whom live in poverty?

The bar chart displays the comparison of the total estimated youth poverty and the state children population across various states in the United States. The x-axis represents the state postal codes, while the y-axis shows the population numbers. The blue bars indicate the state children population, while the red bars represent the total estimated youth poverty within those states. Notably, states like Texas (TX), New York (NY), Florida (FL), and Illinois (IL) show higher overall child populations, with significant portions of youth living in poverty. Conversely, states like Vermont (VT) and Wyoming (WY) have relatively smaller child populations and lower youth poverty estimates. This visualization highlights the disparities in youth poverty rates across different states, emphasizing the regions with higher needs for intervention and support.

In [50]:
IF OBJECT_ID('StatePopulationComparisonView', 'V') IS NOT NULL
   DROP VIEW StatePopulationComparisonView;
GO



Commands completed successfully.

Total execution time: 00:00:00.082

In [51]:
CREATE VIEW StatePopulationComparisonView AS
SELECT 
    State_Postal_Code,
    SUM([ChildrenInPoverty]) AS Total_Estimated_Youth_Poverty,
    SUM(Estimated_Population_5_17) AS State_Children_Population
FROM 
    USSchoolDistrict
GROUP BY 
    State_Postal_Code;
GO


Commands completed successfully.

Total execution time: 00:00:00.427

In [52]:
SELECT 
    State_Postal_Code, 
    Total_Estimated_Youth_Poverty,
    State_Children_Population
FROM 
    StatePopulationComparisonView;
GO


(45 rows affected)

Total execution time: 00:00:00.226

State_Postal_Code Total_Estimated_Youth_Poverty State_Children_Population TX 1002891 5553699 KS 63771 515390 PA 279251 1953664 WI 111881 933007 DE 20639 154361 IN 161674 1164954 IL 301563 2039790 NH 12215 189978 MD 114007 996745 DC 17274 85376 MI 259517 1573270 VA 160033 1379417 ME 20702 186154 SC 149973 831988 RI 20166 151212 WA 132429 1220744 MA 113034 993838 NV 78589 515534 NY 520703 2930047 OH 312747 1901125 SD 21655 161072 LA 187532 781673 MO 152728 1011303 NJ 178573 1477654 OR 76942 630914 MN 96252 964018 MT 21947 176107 OK 129880 709475 PR 233738 418780 FL 500989 3189550 IA 59879 538289 VT 8462 87163 ND 15230 132846 ID 38776 350121 GA 301323 1878578 MS 129241 503543 TN 184826 1132716 NC 274783 1698389 NE 43721 352840 WV 55374 263925 WY 11419 98729 HI 24567 217726 NM 77273 349485 KY 141105 741862 UT 51908 698534

In [3]:
WITH PovertyRates AS (
    SELECT 
        State_Postal_Code,
        District_ID,
        School_District,
        
        Estimated_Population_5_17,
        ChildrenInPoverty,
        (ChildrenInPoverty * 1.0 / Estimated_Population_5_17) * 100 AS PovertyRate
    FROM dbo.USSchoolDistrict
    WHERE Estimated_Population_5_17 > 0 -- To avoid division by zero
)
SELECT 
    
    PovertyRate
FROM PovertyRates;

(11218 rows affected)

Displaying Top 5000 rows.

Total execution time: 00:00:00.126

PovertyRate 5.191411216273485 10.635670177987649 12.852805577420442 10.35376639219274 19.842536440046814 15.364371686210768 15.206658202849486 14.156206415620643 15.151255463193076 13.955535989434296 18.869795109054856 14.288476411446249 12.880386364702021 19.830741079597438 8.866299590568968 15.959252971137522 20.23285232383808 13.976388662205538 17.39731437598736 16.496262868424765 21.869742850276374 12.250728862973762 17.692299894232104 28.07953443258972 12.561657349556066 23.433528566692452 10.364911747575274 15.398325458012103 23.071729957805907 15.848896126459255 28.419419197517183 28.24295010845987 18.122903512233624 22.978714669841334 12.864493996569468 25.319148936170212 32.04743534790684 19.792746113989637 23.023791250959324 21.560574948665298 34.20245398773007 28.595248020008334 25.559313215400625 13.980921120114884 22.33986633626192 14.779489959755615 26.08695652173913 14.711632453567939 24.25908952031775 22.461862177801155 24.61139896373057 12.858173572459286 15.363994385089294 15.366383074274594 22.83631946553295 26.434426229508194 30.55322715842414 12.148516107978752 20.73896631823461 13.795795170691092 13.835395373856912 9.340581756565943 11.676741054802761 23.682102227207178 14.446743556455507 17.26623453774967 14.210038469111849 11.184281202329029 13.244936483763006 22.642938576626946 32.50535331905782 10.407156643485187 10.541827250056675 11.605277699160366 5.164382767351515 17.470957912778516 26.64323831836522 27.7027027027027 25.407566902491542 17.40952380952381 15.224211355414047 13.880298344551573 16.50834198132035 25.006587615283266 26.964650891031262 27.77777777777778 22.90484140233723 28.666666666666668 36.79012345679013 28.737765777927777 15.124378109452735 11.59848165331084 12.308194580476657 37.57941550190597 29.6998420221169 29.19261365681581 25.203693644758285 22.151898734177212 13.242574257425744 34.29349941611522 8.901657874905803 7.647208768799388 19.856078989205926 25.490628445424473 18.211180124223603 16.6383701188455 37.63955342902711 14.09700514268428 28.478057889822594 15.24704803393328 20.83435582822086 13.36831584207896 12.998356827812852 26.972157772621806 18.902452834461986 26.3254113345521 24.070021881838073 5.601341837157293 9.982174688057041 24.268660632296996 34.92063492063492 18.71436764160173 25.30612244897959 9.066411610595916 24.775068457426 31.343974092065697 7.691801985405299 15.265017667844525 26.43781993375489 9.225008510155453 21.175224986765485 32.87671232876712 13.44178082191781 16.83289857296302 10.344041922989291 19.23290766912978 4.012971220105391 31.456953642384107 30.647482014388487 31.549520766773163 38.866285083787325 13.585305545743553 34.84380363886028 33.368255631220535 29.295774647887324 9.163520185884403 23.674698795180724 27.220766849249824 28.37640175524135 17.486522911051214 7.065857414620889 13.986641411623966 3.559315716894637 4.525288376220053 10.67724222086638 19.62303124193132 12.512049354154616 18.77096471219643 18.921389396709323 16.46090534979424 18.693467336683415 13.877324451845684 29.611752360965372 20.731247644176403 9.33617735727523 14.124583440143553 15.39297971388766 34.493346980552715 19.425884682009404 9.523809523809524 19.310344827586206 18.25242718446602 10.737833594976452 15.010739490641301 26.68428005284016 9.247921045417161 17.17638691322902 26.731980405878236 7.926829268292683 28.52841550621001 33.41307814992025 32.74111675126903 14.629777070063692 18.501478803812027 25.87492828456684 25.648508430609596 11.940972745068514 22.069553681417606 24.550359712230215 19.2 13.828061638280618 13.540229885057473 33.12061263560944 13.977104874446086 13.954331279449075 27.715030408340574 25.28592739930383 28.369905956112852 26.144801980198018 27.346938775510203 30.75363298411626 13.030496907656216 28.809523809523807 14.66089466089466 17.06767975235683 26.438104003681545 17.191186289784106 5.143671857863566 17.074164629176856 8.417224323537969 25.41867400779995 38.51851851851852 13.585291113381 21.188279978296258 10.188784008883953 19.08667972575

In [5]:
-- Calculate the poverty rate and join with median household income
WITH PovertyRates AS (
    SELECT 
        usd.State_Postal_Code,
        usd.District_ID,
        usd.School_District,
        usd.Estimated_Population_5_17,
        usd.ChildrenInPoverty,
        (usd.ChildrenInPoverty * 1.0 / usd.Estimated_Population_5_17) * 100 AS PovertyRate,
        inc.Median_Household_Income
    FROM 
        dbo.USSchoolDistrict usd
    JOIN 
        dbo.[2022_income_Estimate] inc
    ON 
        usd.State_Postal_Code = inc.Postal_Code
    WHERE 
        usd.Estimated_Population_5_17 > 0 -- To avoid division by zero
)

-- Select the relevant columns for the scatter plot
SELECT 
    Median_Household_Income,
    PovertyRate
FROM 
    PovertyRates;


(1013318 rows affected)

Displaying Top 5000 rows.

Total execution time: 00:00:12.408

Median_Household_Income PovertyRate 81933 5.191411216273485 71866 5.191411216273485 85427 5.191411216273485 81188 5.191411216273485 81933 10.635670177987649 71866 10.635670177987649 85427 10.635670177987649 81188 10.635670177987649 81933 12.852805577420442 71866 12.852805577420442 85427 12.852805577420442 81188 12.852805577420442 81933 10.35376639219274 71866 10.35376639219274 85427 10.35376639219274 81188 10.35376639219274 81933 19.842536440046814 71866 19.842536440046814 85427 19.842536440046814 81188 19.842536440046814 81933 15.364371686210768 71866 15.364371686210768 85427 15.364371686210768 81188 15.364371686210768 81933 15.206658202849486 71866 15.206658202849486 85427 15.206658202849486 81188 15.206658202849486 81933 14.156206415620643 71866 14.156206415620643 85427 14.156206415620643 81188 14.156206415620643 81933 15.151255463193076 71866 15.151255463193076 85427 15.151255463193076 81188 15.151255463193076 81933 13.955535989434296 71866 13.955535989434296 85427 13.955535989434296 81188 13.955535989434296 81933 18.869795109054856 71866 18.869795109054856 85427 18.869795109054856 81188 18.869795109054856 81933 14.288476411446249 71866 14.288476411446249 85427 14.288476411446249 81188 14.288476411446249 81933 12.880386364702021 71866 12.880386364702021 85427 12.880386364702021 81188 12.880386364702021 81933 19.830741079597438 71866 19.830741079597438 85427 19.830741079597438 81188 19.830741079597438 81933 8.866299590568968 71866 8.866299590568968 85427 8.866299590568968 81188 8.866299590568968 81933 15.959252971137522 71866 15.959252971137522 85427 15.959252971137522 81188 15.959252971137522 99897 20.23285232383808 99897 20.23285232383808 69287 13.976388662205538 58783 13.976388662205538 67375 13.976388662205538 65521 13.976388662205538 57664 13.976388662205538 74899 13.976388662205538 78814 13.976388662205538 49233 13.976388662205538 77851 13.976388662205538 51295 13.976388662205538 48482 13.976388662205538 60749 13.976388662205538 64601 13.976388662205538 63482 13.976388662205538 46459 13.976388662205538 103757 13.976388662205538 64038 13.976388662205538 82537 13.976388662205538 78309 13.976388662205538 80296 13.976388662205538 72129 13.976388662205538 63782 13.976388662205538 76713 13.976388662205538 65933 13.976388662205538 66427 13.976388662205538 61941 13.976388662205538 80678 13.976388662205538 66952 13.976388662205538 79459 13.976388662205538 90808 13.976388662205538 76663 13.976388662205538 50176 13.976388662205538 61560 13.976388662205538 55147 13.976388662205538 49514 13.976388662205538 46141 13.976388662205538 72176 13.976388662205538 54517 13.976388662205538 69462 13.976388662205538 50625 13.976388662205538 56900 13.976388662205538 50775 13.976388662205538 67637 13.976388662205538 71607 13.976388662205538 48572 13.976388662205538 55308 13.976388662205538 60582 13.976388662205538 51954 13.976388662205538 74091 13.976388662205538 50118 13.976388662205538 52180 13.976388662205538 45628 13.976388662205538 58178 13.976388662205538 50559 13.976388662205538 55909 13.976388662205538 45484 13.976388662205538 54060 13.976388662205538 47097 13.976388662205538 40587 13.976388662205538 70203 13.976388662205538 61634 13.976388662205538 73336 13.976388662205538 70834 13.976388662205538 46766 13.976388662205538 65040 13.976388662205538 52534 13.976388662205538 84364 13.976388662205538 81821 13.976388662205538 69287 17.39731437598736 58783 17.39731437598736 67375 17.39731437598736 65521 17.39731437598736 57664 17.39731437598736 74899 17.39731437598736 78814 17.39731437598736 49233 17.39731437598736 77851 17.39731437598736 51295 17.39731437598736 48482 17.39731437598736 60749 17.39731437598736 64601 17.39731437598736 63482 17.39731437598736 46459 17.39731437598736 103757 17.39731437598736 64038 17.39731437598736 82537 17.39731437598736 78309 17.39731437598736 80296 17.39731437598736 72129 17.39731437598736 63782 17.39731437598736 76713 17.39731437598736 65933 17.39731437598736 66427 17.39731437598736 61941 17.39731437598736 80

## **Conclusion**


### **5.1 Summary of Key Findings**

##### **Population Distribution in Texas:**

- The pie chart showed that Texas has the largest total estimated population among all states. Of the total estimated population of approximately thirty million, about one million children between the ages of 5 to 17 live in poverty. This represents approximately 3.4 percent of the total population of Texas living in poverty.
- The significant portion of the total population (in blue) compared to the smaller segments representing children in poverty (in yellow) and children aged 5-17 (in gray) highlights the broader population dynamics within the state.

##### **Household Income and Poverty Statistics:**

- The statistical data revealed the average median household income to be 63,468 with a standard deviation of  16,308.75, indicating a wide variability in income levels.
- The minimum median household income was 28,972, while the maximum was 167,605, further illustrating the economic disparity.
- Total children in poverty were 6,664,873 with an average of 1,015.99 children per state. The standard deviation of 5,529.49 and a range from 2 to 292,262 children in poverty highlight the uneven distribution of child poverty across states.

##### **School Districts with Highest Child Poverty:**

- The bar chart identified the top school districts with the highest number of children in poverty. The New York City Department of Education led with the highest number, followed by Puerto Rico and the Chicago Public School District 299.
- Other significant districts include Houston Independent School District, Clark County School District, Philadelphia City School District, Dade County School District, Detroit Public Schools Community District, Broward County School District, and Dallas Independent School District.

### **5.2 Recommendations**

##### **Community and Educational Support:**

- Develop community programs that support children and families in high-poverty areas. This includes after-school programs, mentorship opportunities, and access to healthcare and social services.

##### **Focus on High-Poverty Districts:**

- Prioritize funding and support for school districts with the highest numbers of children in poverty. Programs should focus on providing educational resources, nutritional support, and family assistance services.